docs/user/04-metrics.md

```diff
@@ -393,6 +393,14 @@ The relevant metrics are:
 
 ## Limitations
 
+The metric gateway setup is based on the following assumptions:
+
+- Cloud Logging Service as Backend
+- The collector has no autoscaling options and has a limited resource setup of 1 CPU and 1.2 GiB memory.
+- Batching is enabled, and a batch contains up to 1024 metrics/batch.
```

Comment:
That are no assumptions on input the customer has control on, so I would not mention that

Reply:
```suggestion
```